.azure/pipelines/ci.yml

```diff
@@ -46,6 +46,8 @@ variables:
   value: .NETCORE
 - name: PostBuildSign
   value: true
+- name: _UseHelixOpenQueues
+  value: ${{ ne(variables['System.TeamProject'], 'internal') }}
```

Comment:
Double-checked this works by comparing expanded YAML for internal and public CI runs